sds/src/scanner/mod.rs

```diff
@@ -1500,11 +1542,11 @@ mod test {
 
         // Due to the ordering of the scan (alphabetical in this case), the match from "a-match" is not
         // excluded yet, but "z-match" is, so only 1 match is found.
```

Comment:
I think the first part of the comment can be removed, (the updated comment say the contrary of this old part)
```suggestion
```